packages/swingset-theme-hashicorp/src/components/side-nav/category.tsx

```diff
@@ -21,8 +22,8 @@ function Category({
 //Swap this out for already existing heading && Enquire about semantics, https://helios.hashicorp.design/components/application-state uses <div>
 function CategoryHeading({ children }: { children: string }) {
   return (
-    <div className="ss-uppercase ss-text-xs ss-font-semibold ss-leading-6 ss-text-foreground-faint ss-border-b ss-border-faint ss-pb-2">
-      {children.toUpperCase()}
+    <div className="ss-capitalize ss-text-xs ss-font-semibold ss-leading-6 ss-text-primary ss-border-b ss-border-faint ss-pb-2">
+      {children}
```

Comment:
We currently capitalize the entire word but Ben's design uses standard capitalization